packages/dso-toolkit/src/components/card/card.mixins.scss

```diff
@@ -3,46 +3,29 @@
 @use "../../variables/media-query-breakpoints";
 @use "../../components/button";
 
+@use "card.variables" as card-variables;
+
 %card-grid {
-  display: grid;
-  grid-template-columns: 1fr minmax(0, 100%);
+  .dso-card-container {
+    display: grid;
+    grid-template-columns: 1fr minmax(0, 100%);
 
-  .dso-card-heading,
-  .dso-card-content {
-    grid-column: 2 /-1;
+    .dso-card-heading,
+    .dso-card-content {
+      grid-column: 2 /-1;
+    }
   }
 }
 
-%card-grid-addon {
-  grid-row: span 2;
-}
-
 @mixin base() {
-  padding-block: units.$u2 units.$u1;
-  padding-inline: units.$u2;
-  inline-size: 100%;
-
-  &:hover {
+  &:has(.dso-card-heading > a:hover) {
```

Comment:
Wordt deze mixin ook in het Web Component gebruikt? Zo ja, werkt deze selector daar ook?

Reply:
Outdated, er lijkt iets te zijn gedaan. Ik sluit 'm, als het anders moet komt dat bij de review/test weer naar voren